opendebates/templates/opendebates/list_ideas.html

```diff
@@ -81,6 +81,11 @@
           <option value="+votes" {% if sort == "+votes" %}selected{% endif %}>
             {% blocktrans %}Fewest Votes{% endblocktrans %}
           </option>
+          {% if PREVIOUS_DEBATE_TIME %}
+          <option value="-current_votes" {% if sort == "-current_votes" %}selected{% endif %}>
+            {% blocktrans %}Top 30 Since Last Debate{% endblocktrans %}
```

Comment:
Per Tobias' comment elsewhere, I *think* we're not supposed to actually limit this query to 30.  So maybe just "Most Votes Since Last Debate" here?  But this should all be confirmed with Lilia.